proposals/lip-0035.md

```diff
@@ -67,9 +66,9 @@ The state of an account on the Lisk mainnet after applying the block at height `
 
 In order to ensure the uniqueness of the account state, we perform the following additional checks:
 
-* Ensure that an account only has an associated public key if and only if there is at least one outgoing transaction from that account.
+* Ensure that an account has a 20-byte address if there is at least one outgoing transaction from that account and an 8-byte address otherwise.
```

Comment:
Just to be sure: there is no way that someone received a transaction in a 20-bytes address because the new address system comes after the regenesis, right?

also, consider changing 20-byte --> 20-bytes and 8-byte --> 8-bytes address